packages/apps/marmalade-marketplace/src/components/CreateToken/index.tsx

```diff
@@ -59,40 +58,39 @@ function CreateTokenComponent() {
   });
 
   const [guardInput, setGuardInput] = useState({
```

Comment:
wouldn't it be better to have this state remain as `undefined` till the `useEffect` take place?

Reply:
I'd vote to iterate that on the next round, since we'd like to deploy the new changes internally.